### ISSUE TITLE
Permissions for unauthenticated users?

### ISSUE THREAD

**Issue by nyxi** [artipie/artipie#1298]
Hi, I'm in the process of upgrading from 0.28.2 to 0.30.1.

From the wiki I can't figure out how to set default permissions on repos. Ie. allow anyone (without authentication) to download files from a `files` repo.

**Comment by olenagerasimova** [comment]
hi, it's possible only if you do not use permissions policy at all (for all repositories). Just do not add `policy` section in the `artipie.yaml`. In this case any user (authenticated or not) will have access to all repositories

**Comment by nyxi** [comment]
Thanks you @olenagerasimova, just to confirm: **with** policy, it is not possible to make a repo publically available without authentication.

**Without** policy, I can make a repo publically available without authentication but not restrict access?

**Comment by fink-andreas** [comment]
> hi, it's possible only if you do not use permissions policy at all (for all repositories). Just do not add `policy` section in the `artipie.yaml`. In this case any user (authenticated or not) will have access to all repositories

Do you mean the /etc/artipie/artipie.yml file?

If I remove the policy section from it, I get this error:

```
artipie    | Exception in thread "main" com.artipie.ArtipieException: Failed to create artipie auth, storage is not configured
artipie    |    at com.artipie.auth.AuthFromStorageFactory.lambda$getAuthentication$1(AuthFromStorageFactory.java:27)
artipie    |    at java.base/java.util.Optional.orElseThrow(Optional.java:403)
artipie    |    at com.artipie.auth.AuthFromStorageFactory.getAuthentication(AuthFromStorageFactory.java:26)
artipie    |    at com.artipie.http.auth.AuthLoader.newObject(AuthLoader.java:59)
artipie    |    at com.artipie.settings.YamlSettings.lambda$initAuth$3(YamlSettings.java:171)
artipie    |    at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
artipie    |    at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
artipie    |    at java.base/java.util.LinkedList$LLSpliterator.forEachRemaining(LinkedList.java:1242)
artipie    |    at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
artipie    |    at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
artipie    |    at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:575)
artipie    |    at java.base/java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:260)
artipie    |    at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:616)
artipie    |    at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:622)
artipie    |    at java.base/java.util.stream.ReferencePipeline.toList(ReferencePipeline.java:627)
artipie    |    at com.artipie.settings.YamlSettings.initAuth(YamlSettings.java:171)
artipie    |    at com.artipie.settings.YamlSettings.<init>(YamlSettings.java:94)
artipie    |    at com.artipie.settings.SettingsFromPath.find(SettingsFromPath.java:51)
artipie    |    at com.artipie.VertxMain.start(VertxMain.java:112)
artipie    |    at com.artipie.VertxMain.main(VertxMain.java:177)
```

**Comment by dgarus** [comment]
This issue is resolved as a part of #1317
